CHANGELOG.adoc

```diff
@@ -139,6 +139,7 @@ Thanks to this capability, a specifier can easily retrieve a variable overriden
 - https://github.com/eclipse-sirius/sirius-web/issues/3096[#3096] [tree] Tree representations can now support filters
 - https://github.com/eclipse-sirius/sirius-web/issues/3158[#3158] [diagram] Add a new tool to apply adjust-size on diagram node.
 - https://github.com/eclipse-sirius/sirius-web/issues/3201[#3201] [diagram] Add support for tools on multiple diagram elements selection (hide, fade and pin).
+- https://github.com/eclipse-sirius/sirius-web/issues/3236[#3236] [diagram] Add a new predicate in `NodeDescription` to specify whether a node is collapsed by default.
```

Comment:
As this PR will only be merged after the 2024.3 release, could you move it in a new 2024.5 section please?

Reply:
done